### ISSUE TITLE
[INSTALL]: Missing optional dependency 'Jinja2'

### ISSUE THREAD

**Issue by tabpier** [pycaret/pycaret#2490]
### Installation check

- [X] I have read the [installation guide](https://pycaret.gitbook.io/docs/get-started/installation).

### Platform

Linux-5.4.188+-x86_64-with-Ubuntu-18.04-bionic

### Installation Method

pip install 

### pycaret Version

2.3.10

### Python Version

3.7.13

### Description

issue with from pycaret.classification import * 

pycaret classification and regression missing optional dependency Jinja2 but the requirement already satisfied Jinja2

"ImportError: Missing optional dependency 'Jinja2'. DataFrame.style requires jinja2. Use pip or conda to install Jinja2"

Try running https://colab.research.google.com/drive/1iVo19vQtD5hk-Kcjuqb2Vg33bMnA1vLu?usp=sharing

**Comment by ngupta23** [comment]
Yes, I noticed this on Colab today morning as well. It was not present until last week. For now, you should do this and restart the kernel

```
pip install -u Jinja2
```

In the 3.0.0 release, we will explicitly include this as can be seen here.

https://github.com/pycaret/pycaret/blob/aa9ec12d4786b762c48e4a8026b80cace8160d8c/requirements.txt#L6

**Comment by erickp1** [comment]
!pip install Jinja2==3.1.2

**Comment by tabpier** [comment]
**!pip install pycaret** 
tensorflow 2.8.0 requires tf-estimator-nightly==2.8.0.dev2021122109, which is not installed.
tensorflow 2.8.0 requires numpy>=1.20, but you have numpy 1.19.5 which is incompatible.
google-colab 1.0.0 requires requests~=2.23.0, but you have requests 2.27.1 which is incompatible.
datascience 0.10.6 requires folium==0.2.1, but you have folium 0.8.3 which is incompatible.
albumentations 0.1.12 requires imgaug<0.2.7,>=0.2.5, but you have imgaug 0.2.9 which is incompatible.

I then installed the correct version of these libraries but **from pycaret.regression import * **  is missing optional dependency Jinja2 so I **!pip install -U Jinja2** which required 
google-colab 1.0.0 requires requests~=2.23.0
flask 1.1.4 requires Jinja2<3.0,>=2.10.1
datascience 0.10.6 requires folium==0.2.1

But the import error persists.

Is there a way I can **from pycaret.regression import * ** in colab? Can I use an older version of pycaret (I have tried but with no success yet)?
